setup.sh

```diff
@@ -5,7 +5,7 @@ set -e
 echo "==> Install TeXLive"
 mkdir -p /tmp/install-tl
 cd /tmp/install-tl
-MIRROR_URL="$(wget -q -S -O /dev/null http://mirror.ctan.org/ 2>&1 | sed -ne 's/.*Location: \(\w*\)/\1/p')"
+MIRROR_URL="$(wget -q -S -O /dev/null http://mirror.ctan.org/ 2>&1 | sed -ne 's/.*Location: \(http\w*\)/\1/p')"
```

Comment:
Could you try and test the following change instead? The thought is that there may be ftp mirrors.

```suggestion
MIRROR_URL="$(wget -q -S -O /dev/null http://mirror.ctan.org/ 2>&1 | sed -ne 's/.*Location: \(\w*\)/\1/p' | head -n 1)"
```

If you the above changes look good to me, please also change the commit message to "setup.sh: only resolve first url redirect".

Reply:
@xu-cheng the `head -n 1` works perfect and I had added a script test step.